sdk/src/main/java/io/customer/sdk/extensions/StringExtensions.kt

```diff
@@ -0,0 +1,9 @@
+package io.customer.sdk.extensions
+
+fun String.getScreenNameFromActivity(): String {
```

Comment:
Previously, we talked about giving customers the ability to provide their own lambda where they can also take in the name of an `Activity` and they can give us the screen name. 

Do we want to add that ability to this PR?

Reply:
Actually, let's not. This is a production hot fix release. We should leave this PR as a bug fix, only. 

Let's create a ticket and merge this as-is.